packages/maskbook/src/extension/background-script/EthereumServices/nonce.ts

```diff
@@ -30,7 +30,9 @@ class NonceManager {
                     this.lock()
                     callback(null, await getTransactionCount(this.address))
                 } catch (e) {
-                    callback(e)
+                    if (e instanceof Error) {
+                        callback(e)
```

Comment:
This change actually changes the semantics and might break the code. Please always re-throw the error